src/org/usfirst/frc4904/robot/commands/FloorioSecureGear.java

```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.FloorIO;
+import org.usfirst.frc4904.standard.commands.SingleOp;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class FloorioSecureGear extends CommandGroup {
+	public FloorioSecureGear() {
+		setTimeout(FloorIO.SECURE_GEAR_TIME);
+		requires(RobotMap.Component.floorIO);
+		addParallel(new SingleOp(() -> RobotMap.Component.floorIO.piston.set(FloorIO.RAISED)));
```

Comment:
This is fine for now, but we should eventually make a generalized `SolenoidSwitch` command or something.